frontend/src/common/components/MentoringApplicationStatus/MentoringApplicationStatus.tsx

```diff
@@ -6,85 +6,60 @@ import type { myTheme } from '../../types/theme';
 
 interface MentoringApplicationStatusProps {
   status: StatusType;
+  type?: 'CREATED' | 'PARTICIPATED';
 }
 
 const STATUS_DESCRIPTION = {
   [StatusTypeEnum.PENDING]: {
     VALUE: '승인대기',
-    EMOTICON: '⏳',
   },
   [StatusTypeEnum.APPROVED]: {
     VALUE: '승인됨',
-    EMOTICON: '✅',
   },
   [StatusTypeEnum.COMPLETE]: {
     VALUE: '완료됨',
-    EMOTICON: '🎉',
   },
   [StatusTypeEnum.REJECTED]: {
     VALUE: '거절됨',
-    EMOTICON: '❌',
+  },
+} as const;
+
+const STATUS_PARTICIPATED_DESCRIPTION = {
+  [StatusTypeEnum.PENDING]: {
+    VALUE: '예약신청',
+  },
+  [StatusTypeEnum.APPROVED]: {
+    VALUE: '예약확정',
+  },
+  [StatusTypeEnum.COMPLETE]: {
+    VALUE: '완료됨',
+  },
+  [StatusTypeEnum.REJECTED]: {
+    VALUE: '거절됨',
```

Comment:
p5: 참여한 멘토링 관련된 데이터 추가 좋은데요~ 👍

type prop으로 분기 처리하니까 개설한 멘토링 데이터와 응집되게 모아도 괜찮을 거 같은데 어때?

```suggestion
const STATUS_DESCRIPTION = {
  CREATED: {
    [StatusTypeEnum.PENDING]: {
      VALUE: '승인대기',
    },
    [StatusTypeEnum.APPROVED]: {
      VALUE: '승인됨',
    },
    [StatusTypeEnum.COMPLETE]: {
      VALUE: '완료됨',
    },
    [StatusTypeEnum.REJECTED]: {
      VALUE: '거절됨',
    },
  },
  PARTICIPATED: {
    [StatusTypeEnum.PENDING]: {
      VALUE: '예약신청',
    },
    [StatusTypeEnum.APPROVED]: {
      VALUE: '예약확정',
    },
    [StatusTypeEnum.COMPLETE]: {
      VALUE: '완료됨',
    },
    [StatusTypeEnum.REJECTED]: {
      VALUE: '거절됨',
    },
  },
} as const;
```

Reply:
[feat: 상태 설명 응집하여 간단하게 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/694/commits/c20d81cc6e4eb87dd3ce76f5e838aeb9eca33225)

너무 좋다 훨씬 보기 편해졌어!